### DISCUSSION TITLE
Using ZOOM with SEB

### DISCUSSION THREAD

**Discussion by vanie76** [SafeExamBrowser/seb-win-refactoring#130] [Q&A]
I integrated ZOOM with SEB version 3.1.0 . Run 40 over e-assessment last semester via MOODLE.
When I downloaded the new version and ran the same config file this semester, the ZOOM is not launching. Please help

**Comment by dbuechel** [comment]
Zoom is as of version 3.2.0 by default a prohibited application. In order to use Zoom, make sure to disable the respective entry under "Applications" -> "Prohibited Processes" in the configuration tool: https://safeexambrowser.org/windows/win_usermanual_en.html#ProhibitedProcessesSection

**Reply by cienciascontic** [reply]
Hi! I'm using SEB 3.3 under Windows. I have disabled Zoom entry under "Applications" -> "Prohibited Processes" in the configuration tool and also established the current configuration in the "Permitted processes" tab:
![image](https://user-images.githubusercontent.com/6078186/129432762-f6039091-eaed-4903-9ca7-1e647ec417d9.png)

I save my configuration .seb file to "open an exam", I loaded that .seb file in the Moodle quiz, then I start Zoom in a different computer as a student and try to do the quiz but SEB doesn't allow me to use Zoom.
What am I doing wrong?
Thanks a lot.
